admin/includes/init_includes/init_sanitize.php

```diff
@@ -249,11 +274,17 @@
 
     $configuration_value = zen_db_prepare_input($_POST['configuration_value']);
     // See if there are any configuration checks
-    $checks = $db->Execute("SELECT val_function FROM " . TABLE_CONFIGURATION . " WHERE configuration_id = " . (int)$cID);
-}
+    $checks = $db->Execute("SELECT configuration_key, val_function FROM " . TABLE_CONFIGURATION . " WHERE configuration_id = " . (int)$cID);
 
-if (!(!$checks->EOF && $checks->fields['val_function'] != NULL)) {
-    $group = array_merge($group, array('configuration_value'));
+    if (!$checks->EOF && $checks->fields['val_function'] != NULL) {
+      $skip_group = array('configuration_value' => array('sanitizerType' => 'NULL_ACTION', 'method' => 'post'));
+      $sanitizer->addComplexSanitization($skip_group);
+    } else if (!$checks->EOF && in_array($checks->fields['configuration_key'], $configs_with_special_characters)) {
+      $skip_group = array('configuration_value' => array('sanitizerType' => 'NULL_ACTION', 'method' => 'post'));
+      $sanitizer->addComplexSanitization($skip_group);

```

Comment:
Since the payload of both the `if` and `elseif` here are identical, does it make sense to combine them?

Reply:
OK.